src/pykoala/data_container.py

```diff
@@ -3,17 +3,255 @@
 reduction process
 """
 
+from astropy.io import Header
 import numpy as np
 
 from pykoala.exceptions.exceptions import NoneAttrError
 
+class LogEntry(object):
+    """Log information unit.
+    
+    Description
+    -----------
+    This class represents a unit of information stored in a log.
+
+    Attributes
+    ----------
+    - title: (str)
+        Title of the entry.
+    - comments: (str)
+        List of strings the information to be stored.
+
+    Methods
+    -------
+    - to_str:
+        Return a string containing the information of 
+        the entry.
+    """
+    def __init__(self, title, comments, tag=None) -> None:
+        self.title = title
+        self.comments = comments
+        self.tag = tag
+    
+    @property
+    def comments(self):
+        return self._comments
+    
+    @comments.setter
+    def comments(self, comments):
+        if isinstance(comments, list):
+            self._comments = comments
+        elif isinstance(comments, str):
```

Comment:
What's the rationale behind this?
For example, "Some interesting comment" would be split into ["Some", "interesting", "comment"]
I'd expect that the default behaviour was
`self._comments = [comments]`